README.md

```diff
@@ -50,3 +56,9 @@
 
 ## Advance ML-on-Code awareness
 
+### App team activities
+
+- [ ] Speak at at least 1 conference
+- [ ] Write at least 1 blog post
```

Comment:
This should be an OKR of the DevRel tasks of publishing blog post, if there is any (I guess yes)

Reply:
:+1: Will be happy to move it there! 
Was just trying to find a way to contribute to that objective.

Reply:
@campoy I thought this might help to declare the intentions and set the right expectations, but now will remove it, as seems to be requested by @mcuadros